charts/kube-prometheus-stack/values.yaml

```diff
@@ -3795,6 +3795,11 @@ prometheus:
     # Set to 'false' to disable global sample_limit. or set to a number to override the default value.
     sampleLimit: false
 
+    #EnforcedKeepDroppedTargetsLimit defines on the number of targets dropped by relabeling that will be kept in memory. 
+    #The value overrides any spec.keepDroppedTargets set by ServiceMonitor, PodMonitor, Probe objects unless spec.keepDroppedTargets 
+    #is greater than zero and less than spec.enforcedKeepDroppedTargets. 0 means no limit.
+    enforcedKeepDroppedTargets: 10
```

Comment:
Please use 0 as default value here to avoid a breaking change.

Reply:
Done